hysds/job_worker.py

```diff
@@ -1059,16 +1061,17 @@ def handler(signum, frame):
         image_url = job.get("container_image_url", None)
         image_mappings = job.get("container_mappings", {})
         runtime_options = job.get("runtime_options", {})
+
+        container_engine = container_engine_factory("docker")
```

Comment:
```suggestion
        container_engine = container_engine_factory(app.conf.get("CONTAINER_ENGINE", "docker"))
```
@DustinKLo: we should also update how we configure the container engine (docker vs. podman vs. singularity). My thinking is that we add a new `celeryconfig.py` parameter called `CONTAINER_ENGINE` (e.g. `CONTAINER_REGISTRY`: https://github.com/hysds/hysds/blob/develop/configs/celery/celeryconfig.py.tmpl#L125) and the above code suggestion will resolve that. If it's undefined in celeryconfig.py then it will use docker by default.

Reply:
Here's an example of setting a parameter with a default value in a jinja template:
https://github.com/hysds/hysds/blob/develop/configs/celery/celeryconfig.py.tmpl#L82
We'd probably want something like:
```
CONTAINER_ENGINE = "{{ CONTAINER_ENGINE or 'docker' }}"
```

We'd also need to update sdscli to populate the `CONTAINER_ENGINE` parameter for us under `~/.sds/config`: https://github.com/sdskit/sdscli/blob/326dd428b67136ac727c50c65c93425c4ddac034/sdscli/adapters/hysds/configure.py#L163-L164